data/examples/south_africa_gravity.py

```diff
@@ -0,0 +1,45 @@
+"""
+Land Gravity Data from South Africa
+===================================
+
+Land gravity survey performed in January 1986 within the boundaries of the Republic of
+South Africa. The data are stored in a :class:`pandas.DataFrame` with columns:
+longitude, latitude, elevation (above sea level) and gravity(mGal). See the
+documentation for :func:`harmonica.datasets.fetch_south_africa_gravity` for more
+information.
```

Comment:
Might be worth mentioning the data source and copyright notice from the docstring here as well.

Reply:
Done!